config.defaults.ini

```diff
@@ -12,8 +12,8 @@ IP: 0.0.0.0
 Useragent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10) AppleWebKit/600.1.25 (KHTML, like Gecko) Version/8.0 Safari/600.1.25
 
 [Cache]
```

Comment:
Because the cache size is now per site, so a value of 150 would be a cache of 4200 images